draft-ietf-lake-edhoc.md

```diff
@@ -711,42 +708,74 @@ The definition of Expand depends on the EDHOC hash algorithm of the selected cip
 
 where L = 8*length, the output length in bits.
 
-The keys, IVs and MACs are derived as follows:
+The keys, IVs, salts and MACs are derived as follows (IVs are only used if the EDHOC AEAD algorithm uses IV):
+
+* KEYSTREAM_2 and SALT_3e2m is derived using the transcript hash TH_2 and the pseudorandom key PRK_2e.
+* MAC_2 is derived using context_2, see {{asym-msg2-proc}}}, and the pseudorandom key PRK_3e2m.
+* K_3, IV_3 and SALT_4e3m are derived using the transcript hash TH_3 and the pseudorandom key PRK_3e2m.
+* MAC_3 is derived using context_3, see {{asym-msg3-proc}} and the pseudorandom key PRK_4e3m.
+* PRK_out, K_4 and IV_4 are derived using the transcript hash TH_4 and the pseudorandom key PRK_4e3m.
+
```

Comment:
The above bullet list can be deleted I think. It is replaced by the table below.